### ISSUE TITLE
[bug] unknown flag: --skip-crds when helmfile repos

### ISSUE THREAD

**Issue by kfirfer** [helmfile/helmfile#713]
### Helmfile Version

0.151.0

### Helm Version

v3.10.3

### Bug description

Hello

When I try to run this:
```
helmfile repos -f helmfile.yaml
```

and the helmfile has :
```
helmDefaults:
  args:
    - "--skip-crds"
```
 Im getting this error:
 ```
 Error: unknown flag: --skip-crds
 ```
 
 How can I use helmfile repos command without removing args of skip-crds ? 
 thanks !

**Comment by yxxhero** [comment]
see: https://github.com/helmfile/helmfile/blob/main/docs/proposals/towards-1.0.md#remove-the---args-flag-from-the-helmfile-command

**Comment by travisghansen** [comment]
Yeah. That’s been discussed a bit. I would just make this a request to formally allow a new flag on a per release basis which controls the behavior for ‘normal’ sync operations and also for template operations as well.

**Comment by yxxhero** [comment]
@travisghansen if you have ideas. PR is welcome.

**Comment by travisghansen** [comment]
Golang is unfortunately not my forte but I can provide some guidance :)

**Comment by yxxhero** [comment]
@travisghansen Thanks very much. ideas are welcome.

**Comment by travisghansen** [comment]
Still needs to be explored.

**Comment by yxxhero** [comment]
@travisghansen any updates? you can do it like this PR: https://github.com/helmfile/helmfile/pull/959, if you don't have free time. please tell me.

**Comment by luarx** [comment]
Any updates here? This would be really helpful 🙏
